### ISSUE TITLE
bottom fishing 

### ISSUE THREAD

**Issue by quattrob5a4** [DarkLord76865/Autofish-Fishing-Planet#1]
update coming any time soon ?

**Comment by DarkLord76865** [comment]
Hi,
thank you for showing interest in my project. Unfortunately, there is no plan when, if ever, bottom fishing will be implemented. It is not that I don't want to do it, it is just that I don't know how. Currently, bot works by taking screenshots and analysing what is visible on the screen (it analyses certain pixels). With spin fishing this works great because one is always holding a rod in hand and all the information about the rod is already on the screen. The same can't be said with multi rod bottom fishing. Bot could be made to work with the rod that is currently being held in hand, but what about other rods that are placed on the rod stand? I don't know how to gather information about them, and fishing with only one bottom rod being held doesn't seem very profitable to me. I understand that bottom fishing is necessary for advancing at higher xp levels (I'm currently level 48 and spin fishing is very very slow to gather xp). I just don't have enough time to devote to develop it further. For bottom fishing to work correctly other things should also be implemented (like custom time warps) so there is a lot of work there. If you can contribute, or know someone who can, feel free to open Pull Request.

TL;DR No, there is no plan for bottom fishing to be implemented in the forseeable future. Sorry!

Regards

**Comment by lordnitorrinco** [comment]
I don't know if you can catch the audio output with python but that can be a way to check the stands with alarm, since they have diferent sounds for each slot.

**Comment by amamic1803** [comment]
I didn't think of that, but that is certainly something I will look into. It doesn't seem to be an easy thing to accomplish, but since bottom fishing is so important after like lvl 50, it might be worth the effort.

**Comment by quattrob5a4** [comment]
waiting brate
